### DISCUSSION TITLE
Downsampling

### DISCUSSION THREAD

**Discussion by aliseena1** [flatironinstitute/CaImAn#1205] [Q&A] [answered]
Hello, we have previously been using the Inscopix software to run CNMFE on our videos. However, we are unable to analyze 20 Hz videos as the software requires us to downsample down to 10. Would the CaImAn toolbox and associated CNMFE allow for 20 Hz videos, or would a similar issue occur? Thank you!

**Comment by EricThomson** [comment, marked as answer]
It will work with any sampling rate. You should be able to use the demo_online_cnmfE.ipynb notebook, and `fr` (frame rate) is just another parameter.

That said, I would be really surprised if they didn't provide an option to analyze at the frame rate of your actual acquisition. I would talk to them. Obviously, I would encourage you to use Caiman :smile: , but I would double check about that sampling frequency thing that would be a very unusual limitation -- sometimes they hide options behind extra clicks.
